pkg/security/tests/module_tester_linux.go

```diff
@@ -1301,12 +1301,10 @@ func (tm *testModule) StartADocker() (*dockerCmdWrapper, error) {
 func (tm *testModule) GetDumpFromDocker(dockerInstance *dockerCmdWrapper) (*activityDumpIdentifier, error) {
 	dumps, err := tm.ListActivityDumps()
 	if err != nil {
-		_, _ = dockerInstance.stop()
```

Comment:
Any particular reason for this ? Could we fail to list activity dumps and yet still have the docker container running ? 🤔